File: 1011.py
from math import ceil
from sys import stdin

input = stdin.readline


def solve(diff):
    n = ceil((diff ** 0.5) - 1)
    if n * (n + 1) < diff:
        return 2 * n + 1
    else:
        return 2 * n


if __name__ == '__main__':
    tc = int(input())
    for _ in range(tc):
        x, y = map(int, input().split())
        print(solve(y - x))

File: 1011.js
const fs = require('fs')

function solve() {
    let input = fs.readFileSync("/dev/stdin").toString().split("\n");
    let inputIdx = 0

    const TC = parseInt(input[inputIdx++])
    let n = 0, diff = 0

    for (let tc = 0; tc < TC; tc++){
        const [x, y] = input[inputIdx++].split(' ').map(v => +v)
        diff = y - x
        n = Math.ceil((diff ** 0.5) - 1)
        if (n * (n + 1) < diff) console.log(2 * n + 1)
        else console.log(2 * n)
    }
}

solve()
